src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -315,7 +335,42 @@ class EduSharingConnector {
 
 			return new EduSharingResponse(parsed);
 		} catch (err) {
-			logger.error('Edu-Sharing failed search ', err.message);
+			logger.error(`Edu-Sharing failed search: ${err.message}`);
+			return Promise.reject(err);
+		}
+	}
+
+	async getPlayerForNode(nodeUuid) {
+		try {
+			const url = `${ES_ENDPOINTS.RENDERER}${nodeUuid}`;
+			const options = {
```

Comment:
The option is build multiple times and should be outsource in a additional method.